src/AxePage.ts

```diff
@@ -51,7 +51,7 @@ function getOptions(options: RunOptions, disabledRules: string[] = []) {
 function addPromiseQueue() {
   type QueuedPromise<T> = {
     promiseCreator: () => Promise<T>;
-    resolve: (value?: T | PromiseLike<T>) => void;
+    resolve: (value: T | PromiseLike<T>) => void;
```

Comment:
Seems that the promise types have been tweaked in the updated typescript version.